strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/test/java/org/carlspring/strongbox/artifact/coordinates/NpmArtifactCoorinatesTest.java

```diff
@@ -0,0 +1,42 @@
+package org.carlspring.strongbox.artifact.coordinates;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import org.junit.Test;
+
+/**
+ * @author sbespalov
+ *
+ */
+public class NpmArtifactCoorinatesTest
```

Comment:
From what I can tell, there also needs to be a test that checks for NPM coordinates that don't have a scope, right?

Reply:
it checked [here](https://github.com/strongbox/strongbox/pull/383/files#diff-ce4d9dbed3e23fadf51f2267a95ada90R18)

Reply:
Ah, great! I seem to have missed that! Thanks! :)